src/test/rustdoc-gui/search-filter.goml

```diff
@@ -14,4 +14,4 @@ press-key: "Enter"
 // Waiting for the search results to appear...
 wait-for: "#titles"
 // We check that there is no more "test_docs" appearing.
-assert-false: "#results .externcrate"
+// assert-false: "#results .externcrate"
```

Comment:
Not sure how we want to handle this. @GuillaumeGomez

Reply:
If the result appears, then it means the search filter is broken. Try to do on your browser what it's doing in this and you should see pretty quickly what's wrong. If you can't figure it out, I can take a look in the next days.

Please revert this change in any case.

Reply:
Reverted.

Reply:
Nice!